src/main/java/seedu/address/storage/JsonAdaptedQuestion.java

```diff
@@ -39,7 +39,8 @@ public JsonAdaptedQuestion(Question source) {
     public Question toModelType() throws IllegalValueException {
 
         if (description == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Description.class.getSimpleName()));
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    Description.class.getSimpleName()));
```

Comment:
I was hesitating to wrap the line as it was only 4 characters longer than 120 characters. But this looks good too, thank you.